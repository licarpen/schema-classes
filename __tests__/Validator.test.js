describe('validator module', () => {
  describe('Validator class', () => {
    it('validate method can take an object and return a fields value', () => {
      expect(isNumber(3)).toBeTruthy();
    });

    it('validate method can take an object and throw an error', () => {
      expect(isString('hi')).toBeTruthy();
    });
  });
});
