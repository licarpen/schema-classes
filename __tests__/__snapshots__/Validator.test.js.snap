// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`validator module validate method takes an object and returns a fields value 1`] = `undefined`;
